core/domain/user_services.py

```diff
@@ -118,10 +145,12 @@ def get_user_settings_from_username(username):
     if user_model is None:
         return None
     else:
-        return get_user_settings(user_model.id)
+        return get_user_settings(user_model.id, strict=False)
```

Comment:
Given that the user_model is not None, why do you need strict=False?

Reply:
Actually I changed the default value of strict for `get_user_settings` from `False` to `True`, reason:

There are many places in the codebase where `get_user_settings` is called without `strict` keyword argument and Its returned value is treated as non-None value ( like returned value cannot contain none ). Due this situation MyPy throwing more than 120+ errors. So, to fix the errors I changed the default value.

Also, all the call sites are fixed accordingly so that nothing will break.

Reply:
Similarly, this call site is fixed by adding `strict=false`. Because previously it was using strict as false

Reply:
I'm ok with changing the default behaviour.

But I'm asking a logical question here -- in what case could this return None anyway?